From cd301edd32c129b23ddcc76a04c281fac8eaaa74 Mon Sep 17 00:00:00 2001
From: Mario Schuknecht <mario.schuknecht@dresearch-fe.de>
Date: Sun, 7 Mar 2021 09:18:41 +0100
Subject: [PATCH]  mke2fs.conf: define a short proceed_delay of 1 sec,
 HYP-14823

Beginning with this version mke2fs asks the user for confirmation if
block device or image file contains an existing file system image.
To avoid the necessary interaction in our scripts we define a short
proceed delay of just a second after which mke2fs continues
automatically.

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 misc/mke2fs.conf.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index 01e35cf8..a72e0cba 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -45,3 +45,6 @@
 	     blocksize = 4096
 	     inode_size = 128
 	}
+
+[options]
+	proceed_delay = 1
-- 
2.26.2

